src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/Program.cs

```diff
@@ -1,24 +1,12 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Company.Application1;
 
-namespace Company.Application1
-{
-    public class Program
+using IHost host = Host.CreateDefaultBuilder(args)
```

Comment:
```suggestion
IHost host = Host.CreateDefaultBuilder(args)
```

I'd remove the using here. RunAsync will dispose (asynchronously).

Reply:
I accepted the suggestion, even though I'm not a fan of it. It reminds me of the `public static IHostBuilder CreateHostBuilder(string[] args)` in that it's "magical", and does more than it _should_. I assume that other developers would see this and ask, "why are we not wrapping this `IDisposable` instance in a `using`?". In other words, it's not obvious that `host.RunAsync()` manages the disposable of itself.